Formula/gromacs.rb

```diff
@@ -25,7 +25,10 @@ class Gromacs < Formula
   depends_on "gcc" # for OpenMP
   depends_on "openblas"
```

Comment:
Note that OpenBLAS uses `libgomp`, so you'll be mixing OpenMP implementations here.

Reply:
Hmm, not sure what do about this....

Reply:
> Note that OpenBLAS uses libgomp, so you'll be mixing OpenMP implementations here.

So is it appropriate to just replace `libomp` with `libgomp` below?

Or is it reasonable to entirely leave out the `depends_on "libomp"` entirely to just get transitive installation dependency or does it provide necessary build-time details?

Reply:
I simply don't think we can build that with libomp

Reply:
To be clearer: I think the issue is in the (proposed) votca formula and should be fixed there. We don't want to introduce this mixed-OpenMP linkage.

Reply:
> I simply don't think we can build that with libomp

What is the actual problem with that?  I have seen openblas being built with clang using spack.io before.

Reply:
> To be clearer: I think the issue is in the (proposed) votca formula and should be fixed there. We don't want to introduce this mixed-OpenMP linkage.

Again I am not sure about this either, we build VOTCA on large variety different compiler and library combination without problems. And I of course tried to restrict votca to gcc as well, which led to this error: https://github.com/Homebrew/homebrew-core/pull/129512#issuecomment-1527816716
because boost was build with clang.

Reply:
> > I simply don't think we can build that with libomp
> 
> What is the actual problem with that? I have seen openblas being built with clang using spack.io before.

If I believe:
https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/openblas/package.py#L205-L209
clang+openmp should works for openblas>0.2.19.